### PR TITLE
[Feat]: Filter by category

### PR DESCRIPTION
**Pull request by parthpandyappp**
- User can filter quiz by categories

**note**: _error logs in the app will be replaced by toasts in future commits. Ignore them while reviewing._

### CODE REVIEW

File: src/pages/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              onChange={(e) => setEmail(e.target.value)}
+            />
+          </div>
+          <div className="inp-unit">
+            <label htmlFor="password">Password</label>
+            <input
+              type="password"
+              name="email"
+              placeholder="**********"
+              onChange={(e) => setPass(e.target.value)}
```
Comment (surbhi84) [inline src/pages/Signup.jsx:46]: Just a sugggestion- you can make these inputs controlled components by setting value as the state.

**Review by surbhi84 — APPROVED** [review]
- Liked that you have used firebase.
- Liked the folder structure with the helper functions folder.
- Code looks clean easy to read.
- Don't see any breaking changes good to merge.

File: src/pages/Signup.jsx (hunk 1 above)
Comment (parthpandyappp) [reply src/pages/Signup.jsx:46]: Okay Surbhi, I'll look into it :+1:

**Comment by parthpandyappp** [comment]
Thankyou @surbhi84 for PR review :+1:

File: public/index.html [hunk 2]
```diff
@@ -5,10 +5,34 @@
...
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://jest-quizionnaire.netlify.app/" />
+    <meta property="og:title" content="JestQuiz" />
+    <meta
+      property="og:description"
+      content="https://jest-quizionnaire.netlify.app/"
+    />
+    <meta property="og:image" content="" />
```
Comment (Logan1x) [inline public/index.html:22]: You need to add og image link to show image when sharing on plateforms

**Review by Logan1x — APPROVED** [review]
The Other changes are looking good.

**Comment by parthpandyappp** [comment]
Thanks for the reviews.

**Pull request merged**